Sources/HipMobileUI/Pages/UserRatingPage.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (C) 2017 History in Paderborn App - Universität Paderborn
+//  
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//  
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using PaderbornUniversity.SILab.Hip.Mobile.UI.Navigation;
+using PaderbornUniversity.SILab.Hip.Mobile.UI.ViewModels.Pages;
+
+
+namespace PaderbornUniversity.SILab.Hip.Mobile.UI.Pages 
+{
+    public partial class UserRatingPage : IViewFor<UserRatingPageViewModel> 
+	{
+        public UserRatingPage() 
```

Comment:
Wrong formatting

Reply:
And where is the wrong formatting here? This file was generated automatically

Reply:
The indentation of the curly braces is wrong. You can use the autoformatter and see how it changes the file correctly.

Reply:
This seems to be a VS or github problem. I have looked into this file on my pc and the braces were formatted well.

Reply:
The problem was that you mixed tabs and spaces in this file. I've quickly fixed it.